packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -282,7 +287,14 @@ const handlePasswordInputChange = (
     });
 };
 
-const CreatePassword = ({ password, platform, validatePassword, onSubmit, error_message }: TCFDCreatePasswordProps) => {
+const CreatePassword = ({
+    password,
+    platform,
+    validatePassword,
+    onSubmit,
+    error_message,
+    ...props
```

Comment:
use Destructuring props here
```suggestion
    is_bvi,
    is_real_financial_stp
```